cli/connector/connector.go

```diff
@@ -37,6 +39,17 @@ type Connector interface {
 
 // Connect connects to the tarantool instance according to options.
 func Connect(opts ConnectOpts) (Connector, error) {
+	// It became common that address is longer than 140 symbols(limit).
```

Comment:
On Linux sun_path is 108 bytes length.
Anyway we should check the minimal length and throw an error if  opts.Address > 108.

Reply:
That patch allows us to use opts.Address longer than sun_path limits.

Reply:
Changed comment

Reply:
Suppose this: 
```go
opts.Address = "/tmp/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
opts.Address = "./" + filepath.Base(opts.Address)
```
pts.Address length will be > 108

Reply:
Changed comments prior to your message